sass/_index.scss

```diff
@@ -1,335 +1,424 @@
-.hero {
-    color: #fff;
-    background: linear-gradient(-45deg, #EE7752, #E73C7E, #23A6D5, #23D5AB);
-    animation: Gradient 10s ease infinite;
-    background-size: 400% 400%;
-
-    display: flex;
-    height: 600px;
-    padding-top: 60px;
-    flex-direction: column;
-    justify-content: center;
-    flex: none;
-
-    .background-img {
-        position: absolute;
-        left: 0;
-        top: 60px;
-        right: 0;
-        bottom: 0;
-        height: 600px;
-        background-image: url('/assets/hero-spiro.svg');
-        background-position: 100% 50%;
-        background-size: cover;
-        background-repeat: repeat;
-        background-attachment: scroll;
+#frontpage {
 
+    #hero {
+        height: calc(100vh - 60px);
```

Comment:
You updated some other `60px`es to `95px`, this one is okay at 60px?

Reply:
Solved in 3809be9